reana_client/cli/status.py

```diff
@@ -23,45 +23,51 @@
 
 import logging
 import os
+import sys
 import traceback
 
 import click
 
-from ..config import (default_organization, default_user)
+from ..config import default_user
 
 
 @click.command()
 @click.pass_context
 @click.option(
-    '-u',
-    '--user',
-    default=default_user,
-    help='User who has created the workflow.')
-@click.option(
-    '-o',
-    '--organization',
-    default=default_organization,
-    help='Organization whose resources will be used.')
-def status(ctx, user, organization):
+    '-at',
+    '--access-token',
+    default=os.environ.get('REANA_ACCESS_TOKEN', None),
+    help='Access token of the current user.')
+def status(ctx, access_token):
     """Show current status of the client session."""
     try:
-        click.echo(click.style('User: {}'.format(user), fg='green'))
-        click.echo(click.style('REANA cluster selected: {}'.
-                   format(ctx.obj.client.server_url), fg='green'))
         _ = ctx.obj.client.ping()
-        click.echo(click.style('REANA cluster status: ready', fg='green'))
+        click.echo(click.style('Connected to: {}'.
+                   format(ctx.obj.client.server_url), fg='green'))
         workflow = os.environ.get('REANA_WORKON', None)
-        click.echo(click.style('Workflow selected: {}'.
-                   format(workflow), fg='green'))
-        workflow_status_response = ctx.obj.client.get_workflow_status(
-            user, organization, workflow)
-        click.echo(click.style('Workflow status: {}'.
-                               format(workflow_status_response['status']),
-                               fg='green'))
-
+        if workflow:
+            click.echo(click.style('Working on: {}'.
+                                   format(workflow), fg='green'))
+            try:
+                workflow_status_response = ctx.obj.client.get_workflow_status(
+                    workflow, access_token)
+                click.echo(click.style(
+                    'Workflow status: {}'.
+                    format(workflow_status_response['status']),
+                    fg='green'))
+            except Exception as e:
+                click.echo(
+                    click.style('Could not retrieve workflow status.'
+                                ' Error: {}'.format(str(e)), fg='red'),
+                    err=True)
+                sys.exit(1)
```

Comment:
Probably we should add here a `finally` where we format the user message since if there is an exception thrown while calling the API the message is going to look like this:
```console
$ # Setting wrong REANA_SERVER_URL.
$ reana-client status
Could not retrieve current status. Error: HTTPConnectionPool(host='192.168.99.100', port=31758): Max retries exceeded with url: /api/ping (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x112b2a450>: Failed to establish a new connection: [Errno 61] Connection refused',))
```
Instead of something like (source #115):
```console
$ reana-client status
Connected to: http://192.168.99.100:31758
Working on: workflow.1
Could not retrieve current status. Error: HTTPConnectionPool(host='192.168.99.100', port=31758): Max retries exceeded with url: /api/ping (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x112b2a450>: Failed to establish a new connection: [Errno 61] Connection refused',))
```

Reply:
I refactored the messages, and now `status` returns:
```
1. # correct REANA_SERVER_URL, no workflow selected
Connected to: http://192.168.99.100:30362
No workflow is selected currently.

2. # correct REANA_SERVER_URL, workflow.1 selected
Connected to: http://192.168.99.100:30362
Working on: workflow.1
Workflow status: created

3. # correct REANA_SERVER_URL, non existing workflow selected
Connected to: http://192.168.99.100:30362
Working on: workflow.2
Could not retrieve workflow status. Error: REANA_WORKON is set to workflow.2, but that workflow does not exist. Please set your REANA_WORKON environment variable appropriately.

4. wrong REANA_SERVER_URL, regardless of workflow selected
Could not connect to the selected REANA cluster server at http://192.168.99.100:30363.
Please make sure you have set your REANA_SERVER_URL environment variable correctly.
```